files/sd-app-qubes-gpg-domain.sh

```diff
@@ -1 +1 @@
-export QUBES_GPG_DOMAIN="sd-gpg"
+if [ "$(hostname)" = "sd-app" ]; then export QUBES_GPG_DOMAIN="sd-gpg" && sudo aa-enforce /usr/bin/securedrop-client ; fi
```

Comment:
Rather than `hostname`, what do you think of using `qubesdb-read /name`? That seems to be the Qubesy way to detect what the VM name is, and it's strictly speaking a bit more accurate:

```
$ hostname
sd-dev
$ qubesdb-read /name
sd-dev
$ sudo hostname sd-dev2
$ hostname
sd-dev2
$ qubesdb-read /name
sd-dev
```

Reply:
Good idea, I've implemented these changes in f9f0136